theme.config.ts

```diff
@@ -45,3 +45,34 @@ export const networks: Record<string, Network> = {
  * Layouts in which the footer should not be visible
  */
 export const footerLayoutBlacklist = ['cover', 'intro', 'end'];
+
+export const getSectionTitleGetter = ($slidev: any, rawRoutes: any[]) => () => {
```

Comment:
Zijn er geen types voor vanuit slidev?

Reply:
Nope. I searched the depths of Slidev, but couldn't find them